README.md

```diff
@@ -1335,7 +1335,7 @@ It supports the following Spark deploy modes:
 
 ### Configuring Spark Project
 
-When using Spark backend uncomment the following Spark configuration in the application.conf file
+When using Spark backend uncomment the following Spark configuration in the reference.conf file, which is available under `core/src/resources`
```

Comment:
@cjllanwarne  Thank you for pointing out, it's been a while since i've touched the Cromwell repo. Therefore, not sure about refactoring and moving pieces, if you can help me to make an appropriate configuration change would be nice otherwise i would need to compile and fix it.